src/main/java/org/jboss/forge/arquillian/container/model/Container.java

```diff
@@ -178,6 +178,12 @@ public String getChameleonTarget(String version) {
     }
 
     public boolean isSupportedByChameleon(String version) throws Exception {
+
+        final boolean isProblemWithChameleon = issueWithChameleonForClassloading();
+        if (isProblemWithChameleon) {
+            return !isProblemWithChameleon;
+        }
```

Comment:
Does the method with name issueWithChameleonForClassloading is not denoting why that two containers are excludes?